src/pybind/mgr/dashboard_v2/frontend/src/app/shared/models/health.ts

```diff
@@ -0,0 +1,36 @@
+export class Health {
```

Comment:
We already have on model file that uses the suffix `.model.ts`.
You should rename this file to `health.model.ts` or rename the existing file to `credentials.ts`, for consistency.

Reply:
I change credentials file to credentials.ts.
Angular uses that naming for their models [1].

[1] https://angular.io/tutorial/toh-pt1#create-a-hero-class